prusti-viper/src/encoder/counterexample_translation.rs

```diff
@@ -137,7 +137,7 @@ impl<'ce, 'tcx> CounterexampleTranslator<'ce, 'tcx> {
             def_id,
             silicon_counterexample,
             tcx: encoder.env().tcx(),
-            is_pure: encoder.is_pure(def_id),
+            is_pure: false, // No verified functions are pure. encoder.is_pure(def_id),
```

Comment:
A lot more can be simplified in counterexamples once this is the case. Please add a `TODO` at least.

Reply:
Added. Thanks!